en/Data Structures/Queues/queue.md

```diff
@@ -0,0 +1,49 @@
+# Queue
+
+## Description:
```

Comment:
Please remove the colons from your headings. (Also leave a blank line after headings.)